File: fibnocii/cpp/fibnocii.cpp

#include<iostream>
using namespace std;

void fibo()
{ long input=0,num1=0,num2=1,i=0; // giving default values

  cout<<" Enter the limit for the number of output from the series:- ";
  cin>>input;
  for(i=0;i<input;i++)
  {
    cout<<" num"<<i+1<<": "<<num1+num2<<" "; // Printing next number

    num1=num1+num2;       //   moving to the next number by swap
    num2=num1-num2;
    num1=num1-num2;

    num2= num1+num2;     //  next value updated
  }
}

int main(void)
{
  fibo();       // Fibo Fucntion call
  return 0;
}

File: fibnocii/python/fibnocii.py

def fibo():
    num1,num2,into,i =0,1,0,0
    into=int(input("Enter the limit for the number of output from the series:- "))
    while(i<into):
        print(" num ",i+1,": ",num1+num2," ")

        num1=num1+num2;       #   moving to the next number by swap
        num2=num1-num2;
        num1=num1-num2;
        num2=num1+num2;       # next value updated
        i=i+1

def main():
    fibo()

if __name__ =='__main__':
    main()
